lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -86,7 +88,10 @@ $-separator-color: var(--nx-swatch-indigo-40);
 }
 
 .nx-global-sidebar__logo {
-  height: $-logo-height;
+  // a temporary stop gap to ensure no logos break the header container - matches plaid-villain logo width
+  max-width: 155px;
```

Comment:
> Why 155? I'm not seeing that in sketch. The comment above makes it sound like it's just to match the plaid-villain logo, which doesn't seem like a good basis. Given that the parent is a flex container I bet we can avoid specifying a `max-width` here and just let it shrink to fit the available space. Using that approach, we'd need design to clarify the min space between the logo and the collapse button.

Thanks for this idea. You're right, I used 155px because that was the current width of the plaid-logo to avoid breaking visual tests, which isn't a good enough reason. I'm going to reach out to Jason about the min space between logo & toggle which would resolve the problem of needing some sort of width restriction to prevent all images from increasing to 43px high, and then breaking the width of the header container.

Reply:
> Given that the parent is a flex container I bet we can avoid specifying a `max-width` here and just let it shrink to fit the available space. Using that approach, we'd need design to clarify the min space between the logo and the collapse button.

So I discussed this with Jason and they decided on a 12px padding between the logo & collapse button. Implementing that, the original plaid-logo expands to 43px high, which clearly isn't the desirable effect. 
![Screenshot 2023-04-27 at 10 48 23 AM](https://user-images.githubusercontent.com/89862727/234900052-dfda9e70-9f24-485a-b092-c1035e96ce18.png)
If I set a max-width on the image to 100%, this cleans up the problem when the sidebar is open, but when closed, shrinks the image down to 0px x 0px, and therefore shrinks the height of the header. What are your thoughts on setting a max-width as a calculation of available space instead of a percent? Something like 
`calc($-width-open - calc(2 * $-left-right-spacing) - var(--nx-spacing-3x) - 32px)` ?  
(where var(--nx-spacing-3x) refers to padding between logo & button, and 32px the width of the button?)

Reply:
> Something like
calc($-width-open - calc(2 * $-left-right-spacing) - var(--nx-spacing-3x) - 32px) ?

Unfortunate, but that's probably what we'll have to do.  Initially I was going to say use `max-width:100%` but then change that `max-width` to `unset` when the sidebar is closed, but that doesn't help because then the logo returns to its natural size on collapse and pushes the other content _down_.

I also started to play around with getting the header to overflow off the left side in such a way that it hides the logo when collapsed, but ultimately, because the collapse/expand button also actually overflows the content box, and we need to keep that fully visible, it'd still involve using the width of the button as a magic number making it no better than your solution.

Reply:
> Unfortunate, but that's probably what we'll have to do. Initially I was going to say use `max-width:100%` but then change that `max-width` to `unset` when the sidebar is closed, but that doesn't help because then the logo returns to its natural size on collapse and pushes the other content _down_.
> 
> I also started to play around with getting the header to overflow off the left side in such a way that it hides the logo when collapsed, but ultimately, because the collapse/expand button also actually overflows the content box, and we need to keep that fully visible, it'd still involve using the width of the button as a magic number making it no better than your solution.

I also thought `max-width: unset` would do it, and was disappointed to see that wasn't the case. Not ideal, but updated the max width [092b7eb](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/092b7eb829c85adaef9fae63f80854403b8f976b)